pkg/cli/image_cmd_test.go

```diff
@@ -570,7 +616,7 @@ func TestServerResponse(t *testing.T) {
 		})
 
 		Convey("Test image by digest", func() {
-			args := []string{"imagetest", "--digest", "883fc0c5"}
+			args := []string{"imagetest", "--digest", "9beeea29"}
```

Comment:
Let's have a separate call on these tests. I don't understand why some digests need to be updated while others don't.